src/lib/config.ts

```diff
@@ -1191,6 +1208,10 @@ const platform_defaults = {
 (New-Object System.Net.WebClient).DownloadFile('https://raw.githubusercontent.com/tridactyl/native_messenger/master/installers/windows.ps1', '%TEMP%/tridactyl_installnative.ps1');\
 & '%TEMP%/tridactyl_installnative.ps1' -Tag %TAG;\
 Remove-Item '%TEMP%/tridactyl_installnative.ps1'"`,
+        downloadforbiddenchars: "#%&{}\\<>*?/$!'\":@+`|=",
```

Comment:
Hey @gsbabil, thanks for this PR, I finally got around to looking at it :)

It looks like Linux/OSX permit many more characters in filenames than Windows does, so I have moved the Windows settings to the Windows-only defaults

I've also renamed the settings to match up better with the `downloadsskiphistory` setting in the hope that people will find them easier to find.

Let me know if you have any thoughts - otherwise I think this is good to merge.